games-emulation/pcsx2/pcsx2-1.3.1.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+PLOCALES="ar_SA ca_ES cs_CZ de_DE es_ES fi_FI fr_FR hr_HR hu_HU id_ID it_IT ja_JP ko_KR ms_MY nb_NO pl_PL pt_BR ru_RU sv_SE th_TH tr_TR zh_CN zh_TW"
+
+inherit cmake-utils l10n multilib toolchain-funcs wxwidgets
```

Comment:
I'd personally move cmake-utils to the end of the inherit line, just in case one of the other ones do something funny in the future. But that's debatable.